### ISSUE TITLE
[Content organization] Introduction page doesn't follow heading hierarchy. Headings aren't properly nested, and headings and/or labels aren't relevant to the page. (02.04.1)

### ISSUE THREAD

**Issue by chaseakirby** [department-of-veterans-affairs/va.gov-team#40367]
#### VFS team name
DGIB - My Education Benefits

#### VFS product name
My Education Benefits

#### VFS feature name
My Education Benefits App

#### Point of Contact/Reviewers
Brian DeConinck (@briandeconinck) - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Your page doesn't follow heading hierarchy. Headings aren't properly nested, and headings and/or labels aren't relevant to the page.

### Issue Details
On the Introduction Page (`/apply-for-benefits-form-22-1990/introduction`), the H2 heading level is skipped.

After the H1, each major section of the page should be an H2, with subsections as H3s, then H4s, etc. Heading levels should not be skipped.

Heading levels are announced to screen reader users and those levels are often used for navigation and wayfinding, so skipping an expected heading level can be disorienting.

### Link, screenshot or steps to recreate
Current heading structure:
```
H1: Apply for VA education benefits
          H3: This application is only for the Post-9/11 GI Bill
          H3: Follow these steps to get started
               H4: Check your eligibility
               H4: Gather your information
               H4: Start your application
          H3: Save time--and save your work in progress--by signing in before starting your application
          H3: Begin your application for education benefits [when authenticated]
          H3: Your application is in progress and was last saved on... [when logged in + application in progress]
```

### VA.gov Experience Standard
[Category Number 02, Issue Number 04](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 1.3.1_A
WCAG SC 2.4.6_AA

---

### Platform Recommendation
Recommended heading structure:

```
H1: Apply for VA education benefits
     H2: This application is only for the Post-9/11 GI Bill
     H2: Follow these steps to get started
          H3: Check your eligibility
          H3: Gather your information
          H3: Start your application
     H2: Save time--and save your work in progress--by signing in before starting your application
     H2: Begin your application for education benefits [when authenticated]
     H2: Your application is in progress and was last saved on... [when authenticated + application in progress]
```

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] Close the ticket when the issue has been resolved or validated by your Product Owner. If a team has additional questions or needs Platform help validating the issue, please comment in the ticket.

**Comment by cesaresantiago** [comment]
This is being tracked by MEB team on ticket SA-12822.

**Comment by cesaresantiago** [comment]
Fix developed and tested.
